test/ci-operator-integration/base/expected_files/expected.json

```diff
@@ -1118,8 +1118,9 @@
     "tag": {
       "annotations": null,
       "from": {
-        "kind": "DockerImage",
-        "name": "registry.svc.ci.openshift.org/ocp/4.3@sha256:SHA"
+        "kind": "ImageStreamImage",
+        "name": "4.3@",
```

Comment:
Missing the sha here looks odd?

Reply:
Debugging these tests does not spark a lot of joy.

Tha `sha256:SHA` string was something we harcoded it to when src client != dest client: https://github.com/openshift/ci-tools/pull/881/files#diff-34cb0254875de41c784526c5e3f81ab4L100